<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>19</y>
      <width>341</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>User Id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userIdEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="printerComboBox"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="printNowButton">
         <property name="text">
          <string>Print Now</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>urlEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_urlUpdated(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printerComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>setPrinter(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_updateButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printNowButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_printButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userIdEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_userIdUpdated(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_urlUpdated(QString)</slot>
  <slot>on_printButtonClicked()</slot>
  <slot>setPrinter(QString)</slot>
  <slot>on_updateButtonClicked()</slot>
  <slot>on_userIdUpdated(QString)</slot>
 </slots>
</ui>
